addons/code-analysis/addon.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "Code analysis",
+  "description": "Analyze the code of the Scratch Community Project",
+  "tags": ["analysis", "community", "project"],
+  "version": "1.31.0",
+  "credits": [
+    {
+      "name": "Gandi IDE",
+      "link": "https://github.com/Gandi-IDE"
+    }
+  ],
+  "userscripts": [
+    {
+      "url": "userscript.js",
+      "matches": ["https://scratch.mit.edu/projects/*/"],
+      "runAtComplete": true
+    }
+  ],
+  "userstyles": [
+    {
+      "url": "style.css",
+      "matches": ["https://scratch.mit.edu/projects/*/"]
+    }
+  ],
+  "dynamicEnable": true,
+  "dynamicDisable": true
```

Comment:
I don't think you intentionally set these 2 to true. I think you can set them to false (or remove the properties) for now.